AppHub/AppHub/AHBuild.m

```diff
@@ -58,13 +60,16 @@ - (BOOL)isDefaultBuild
 
 - (NSDictionary *)dictionaryValue
 {
-    return @{
+    NSMutableDictionary *dict = [@{
         AHExportedBuildDataBuildIDKey: _identifier,
         AHExportedBuildDataNameKey: _name,
         AHExportedBuildDataDescriptionKey: _buildDescription,
-        AHExportedBuildDataCreatedAtKey: @(_creationDate.timeIntervalSince1970 * 1000.0),
```

Comment:
that would create a key with `NSNull` as value rather than `nil`. I suppose that works too because on the JS side it gets exported to `null` anyways (rather than `undefined`) which is actually more semantically correct from JS's point of view.